tests/aframe.test.mjs

```diff
@@ -98,29 +103,66 @@ test.describe('Navigation', () => {
 
 test.describe('Prebid', () => {
   test('Ad creative is loaded once bids is no longer null', async ({ page }) => {
-    const banner = page.locator('#banner1 > a-plane');
-    await injectIFrame(page, EXAMPLE_URL, EXAMPLE_IMAGE, '00000000-0000-0000-0000-000000000000');
+    const banner1 = page.locator('#banner1 > a-plane');
+    const banner2 = page.locator('#banner2 > a-plane');
+    const banner3 = page.locator('#banner3 > a-plane');
+    await injectIFrame(page, EXAMPLE_URL, EXAMPLE_IMAGE_MEDIUM_RECTANGLE, '00000000-0000-0000-0000-000000000000');
```

Comment:
Would it be easier to have the ad unit IDs also as constants so you can distinguish between different formats' IDs?